javascript/javascript1/week2/Week2.js

```diff
@@ -0,0 +1,159 @@
+
+// event application 
+
+function getEventWeekday(remainingDay) {
+    const date = new Date();
+    const day = date.getDay();
+    const eventDay = day + remainingDay;
+    const daylist = ["Sunday", "Monday", "Tuesday", "Wednesday ", "Thursday", "Friday", "Saturday"];
+    const message = `Today is ${daylist[day]}.The event is going to happen in ${eventDay} days to come. Which is on ${daylist[eventDay % 7]}`
+
+    console.log(message);
+
+}
+getEventWeekday(14);
+
+/*Weather wear */
+
+function dressAdviser(temperature) {
+    let todaysDress;
+    if (temperature > 0) {
+        if (temperature < 5) {
+            todaysDress = `temperatue can go upto 0 degree so wear a warm cloths`;
+        } else if (temperature < 10) {
+            todaysDress = `temperature can be 5 to 10 degree, so wear warm colthes`;
+        } else if (temperature < 20) {
+            todaysDress = `temperatue can be between 10 to 20 degree.  adviced to wear thin jacket`;
+        } else if (temperature < 30) {
+            todaysDress = `It is a a bit warm today so it is adviced to wear shorts and a t-shirt`;
+        } else {
+            todaysDress = `it very hot today, adviced to seat inside AC or take an sun-umbrella if you need to go outside`
+        }
+    } else {
+        todaysDress = `it is very cold today so it is adviced to  wear a thick jacket,gloves, cap , muffler and warm shoes`;
+    }
```

Comment:
An alternative to avoid the nested if clause would be something like:
```
if (temperature <= 0){
...
} else if (temperature < 5){
..
}else if (temperature < 10){
}...